gdx/src/com/badlogic/gdx/math/Quaternion.java

```diff
@@ -160,40 +160,40 @@ public int getGimbalPole () {
 	/** Get the roll euler angle in radians, which is the rotation around the z axis. Requires that this quaternion is normalized.
 	 * @return the rotation around the z axis in radians (between -PI and +PI) */
 	public float getRollRad () {
-		final int pole = getGimbalPole();
-		return pole == 0 ? MathUtils.atan2(2f * (w * z + y * x), 1f - 2f * (x * x + z * z))
-			: (float)pole * 2f * MathUtils.atan2(y, w);
+		final double pole = getGimbalPole();
```

Comment:
getGimbalPole() hasn't changed, as far as I can see. I don't see a reason to make `pole` a double instead of an int, since it's only checked for equality with int 0.

Reply:
Also, it's still an `int` in `getPitchRad()`.

Reply:
Good catch. I verified that making `pole` a double versus an int has no impact on precision! 👍